code/game/objects/items/storage/fancy.dm

```diff
@@ -501,7 +501,7 @@
 	name = "pickles"
 	desc = "A jar for containing pickles."
 	spawn_type = /obj/item/food/pickle
-	spawn_count = 5
+	spawn_count = 10
```

Comment:
This isn't in the changelog or the body of your PR at all.

Reply:
the recipie requires 10, so figured you should get 10 pickles